ministryofjustice/laa-apply-for-criminal-legal-aid
# [CRIMAPP-1096] Property ownership must be less than 100% if there are other owners

## Description of change
When validating percentage ownership amounts for a property with other owners, the percentage(s) entered are no longer allowed to equal 100% as there are more ownership details to be entered. 

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-1096

## Notes for reviewer

## Screenshots of changes (if applicable)

### Before changes:

### After changes:
<img width="990" alt="Screenshot 2024-08-02 at 11 42 52" src="https://github.com/user-attachments/assets/7eb4b972-9f24-4811-8a25-df57d28a9246">


## How to manually test the feature
Create a property (with or without a partner) and enter a total % ownership that is equal to 100%. Then select yes for other owners and click save and continue. You should see the error message attached above. Then click no for other owners and you should be able to progress. Test for all property types (residential, land and commercial)
